__test__/integration/token_api.spec.ts

```diff
@@ -504,13 +504,18 @@ describe("Token API (Integration Tests)", () => {
                 const tokenSymbolList = await tokenApi.getSupportedTokens();
 
                 // Get a reference from an unsorted array of data we know is correct.
-                const expectedTokenSymbolList = TOKEN_REGISTRY_TRACKED_TOKENS.map((token) => {
-                    return {
-                        symbol: token.symbol,
-                        name: token.name,
-                        numDecimals: token.decimals,
-                    };
-                });
+                const expectedTokenSymbolList = _.filter(
```

Comment:
so this would read cleaner to me chained:

`_.chain().map.().filter().value()`